microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,10 +6,14 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 02/24/2023
+ms.date: 02/09/2024
 ---
 # Test upcoming APIs and features
-<!-- old title: # Switch to a preview channel to test upcoming APIs and features -->
+<!-- old title: 
+# Switch to a preview channel to test upcoming APIs and features
+-->
+
+<!-- todo: update the article to use the new API -->
```

Comment:
update the "Test upcoming APIs and features" article (set-preview-channel.md) to use the new API
@victorhuangwq

Reply:
article, in Files Changed tab, has had extensive modifs.  assuming shows new api; covers API approach incl WinRT.  
closing.